test/Microsoft.Extensions.Options.Test/project.json

```diff
@@ -17,15 +18,12 @@
       ],
       "dependencies": {
         "dotnet-test-xunit": "1.0.0-dev-*",
-        "Microsoft.NETCore.Platforms": "1.0.1-*",
         "System.Dynamic.Runtime": "4.0.11-*"
       }
     },
     "net451": {
-      "frameworkAssemblies": {
-        "System.Runtime": ""
-      },
       "dependencies": {
+        "System.Runtime": "4.1.0-*",
```

Comment:
It's the same issue as Configuration. Guard rail fails saying it can't find a runtime lib for `System.Runtime`.